src/components/MoneyRequestHeader.js

```diff
@@ -111,6 +129,23 @@ const MoneyRequestHeader = (props) => {
                     </View>
                 </View>
             </View>
+            {props.isSingleTransactionView && (
+                <>
+                    <MenuItemWithTopDescription
+                        title={formattedTransactionAmount}
+                        description="Amount • Cash"
+                        titleStyle={styles.newKansasLarge}
+                    />
+                    <MenuItemWithTopDescription
+                        description="Description"
+                        title={transactionDescription}
+                    />
+                    <MenuItemWithTopDescription
+                        description="Date"
```

Comment:
We'll have to follow up and translate these

Reply:
Looks like we already have translation keys for these: [Amount](https://github.com/Expensify/App/blob/f365b889f1be74503de98e481a655c38a4fa05e2/src/languages/en.js#L314), [Cash](https://github.com/Expensify/App/blob/f365b889f1be74503de98e481a655c38a4fa05e2/src/languages/en.js#L315), [Date](https://github.com/Expensify/App/blob/f365b889f1be74503de98e481a655c38a4fa05e2/src/languages/en.js#L62), [Description](https://github.com/Expensify/App/blob/f365b889f1be74503de98e481a655c38a4fa05e2/src/languages/en.js#L136)